deploy/compliance-operator-chart/crds/compliance.openshift.io_compliancescans_crd.yaml

```diff
@@ -74,6 +74,12 @@ spec:
                   generated from the scan, this should match the selector of the MachineConfigPool
                   you want to apply the remediations to.
                 type: object
+              priorityClass:
+                description: Defines the PriorityClass to use for launching scan related
+                  pods, the Name of a desired PriorityClass should be set here, this
+                  is an optional field, if PriorityClass is invalid, not found, or
+                  not set we will ignore this setting.
```

Comment:
I would reword this to remove "we", since we don't use first-person in other parts of the document from what I can tell.

Perhaps:

`this is an optional field, if PriorityClass is invalid or not found, it will be ignored.`